File: update_ids.py
import json

with open('data/specialists.json', 'r', encoding='utf-8') as f:
    data = json.load(f)

for i, item in enumerate(data):
    item['id'] = i + 1

with open('data/specialists.json', 'w', encoding='utf-8') as f:
    json.dump(data, f, ensure_ascii=False, indent=2)

File: update_ids.js
const fs = require('fs');
const data = JSON.parse(fs.readFileSync('data/specialists.json', 'utf8'));
let id = 1;
for (const item of data) {
    item.id = id++;
}
fs.writeFileSync('data/specialists.json', JSON.stringify(data, null, 2));
